src/main/scala/org/broadinstitute/dsde/rawls/webservice/RawlsApiService.scala

```diff
@@ -73,4 +82,29 @@ trait RootRawlsApiService extends HttpService {
       }
     }
   }
+
+  private def serveSwaggerIndex(): Route = {
+    val indexHtml = getResourceFileContents(swaggerUiPath + "/index.html")
+    val jsonFactory = JacksonFactory.getDefaultInstance
+    val clientSecrets = GoogleClientSecrets.load(jsonFactory, new StringReader(clientSecretsJson))
+    complete {
+      HttpEntity(ContentType(`text/html`),
+        indexHtml
+          .replace("your-client-id", clientSecrets.getDetails.getClientId)
```

Comment:
given that all you need is the client id, why not just pass that around instead of the client secret json? Creating a spoofed json for tests is cumbersome, it can't just be random stuff or it won't parse. If it is just an id and empty string will probably work in the tests.